src/supports.js

```diff
@@ -13,3 +14,19 @@ export const removeEventListener = canUseDOM && 'removeEventListener' in window;
 // IE8+ Support
 export const attachEvent = canUseDOM && 'attachEvent' in window;
 export const detachEvent = canUseDOM && 'detachEvent' in window;
+
+// Passive options
+// Inspired by https://github.com/Modernizr/Modernizr/blob/master/feature-detects/dom/passiveeventlisteners.js
+export const passiveOption = () => {
```

Comment:
I also think it's best. Which of following patterns do you think good?

Patter 1: Simple `let` (Probably will be cached)

```javascript
export let passiveOption = false;

try {
  // ...
  passiveOption = true;
} catch (e) {}
```


Pattern 2: memoize

```javascript
export const passiveOption = (() => {
  let cache = null;

  return (() => {
    if (cache != null) return cache;

    let supportsPassiveOption = false;

    try {
      window.addEventListener('test', null, defineProperty({}, 'passive', {
        get() {
          supportsPassiveOption = true;
        },
      }));
    } catch (e) {} // eslint-disable-line no-empty

    cache = supportsPassiveOption;

    return supportsPassiveOption;
  })();
})();
```